Toggl.Droid/Activities/SettingsActivity.cs

```diff
@@ -35,68 +38,88 @@ protected override void InitializeBindings()
+            .Subscribe(durationFormatRow.SetRowData)
+            .DisposedBy(DisposeBag);
+              
+            ViewModel.BeginningOfWeek
+            .Select(beginningOfWeek => new NavigationRow(FirstDayOfTheWeek, beginningOfWeek, ViewModel.SelectBeginningOfWeek))
+            .Subscribe(beginningOfWeekRow.SetRowData)
+            .DisposedBy(DisposeBag);
+              
+            ViewModel.IsGroupingTimeEntries
+            .Select(groupTEs => new ToggleRow(GroupTimeEntries, groupTEs, ViewModel.ToggleTimeEntriesGrouping))
+            .Subscribe(isGroupingTimeEntriesRow.SetRowData)
+            .DisposedBy(DisposeBag);
 
+            
+            // timer defaults section 
             ViewModel.IsManualModeEnabled
-                .Subscribe(manualModeSwitch.Rx().CheckedObserver(ignoreUnchanged: true))
+                .Select(isManualModeEnabled => new ToggleRowWithDescription(ManualMode, ManualModeDescription, isManualModeEnabled, ViewModel.ToggleManualMode))
+                .Subscribe(isManualModeEnabledRowView.SetRowData)
                 .DisposedBy(DisposeBag);
-
-            ViewModel.IsGroupingTimeEntries
-               .Subscribe(groupTimeEntriesSwitch.Rx().CheckedObserver(ignoreUnchanged: true))
-               .DisposedBy(DisposeBag);
-
-            ViewModel.UseTwentyFourHourFormat
-                .Subscribe(is24hoursModeSwitch.Rx().CheckedObserver(ignoreUnchanged: true))
+            
+            ViewModel.SwipeActionsEnabled
+                .Select(areSwipeActionsEnabled => new ToggleRow(SwipeActions, areSwipeActionsEnabled, ViewModel.ToggleSwipeActions))
+                .Subscribe(swipeActionsRow.SetRowData)
                 .DisposedBy(DisposeBag);
-
+            
             ViewModel.AreRunningTimerNotificationsEnabled
-                .Subscribe(runningTimerNotificationsSwitch.Rx().CheckedObserver(ignoreUnchanged: true))
+                .Select(runningTimerNotificationsEnabled => new ToggleRow(NotificationsRunningTimer, runningTimerNotificationsEnabled, ViewModel.ToggleRunningTimerNotifications))
+                .Subscribe(runningTimeEntryRow.SetRowData)
                 .DisposedBy(DisposeBag);
-
+            
             ViewModel.AreStoppedTimerNotificationsEnabled
-                .Subscribe(stoppedTimerNotificationsSwitch.Rx().CheckedObserver(ignoreUnchanged: true))
-                .DisposedBy(DisposeBag);
-
-            ViewModel.DateFormat
-                .Subscribe(dateFormatTextView.Rx().TextObserver())
-                .DisposedBy(DisposeBag);
-
-            ViewModel.BeginningOfWeek
-                .Subscribe(beginningOfWeekTextView.Rx().TextObserver())
-                .DisposedBy(DisposeBag);
-
-            ViewModel.DurationFormat
-                .Subscribe(durationFormatTextView.Rx().TextObserver())
+                .Select(stoppedTimerNotificationsEnabled => new ToggleRow(NotificationsRunningTimer, stoppedTimerNotificationsEnabled, ViewModel.ToggleRunningTimerNotifications))
```

Comment:
Found this bug while testing it...
```suggestion
                .Select(stoppedTimerNotificationsEnabled => new ToggleRow(NotificationsRunningTimer, stoppedTimerNotificationsEnabled, ViewModel.ToggleStoppedTimerNotifications))
```